### ISSUE TITLE
Support rubocop-sorbet

### ISSUE THREAD

**Issue by bmulholland** [codeclimate/codeclimate-rubocop#264]
Now that typechecking is coming in Ruby 3, sorbet will likely become more popular. https://github.com/shopify/rubocop-sorbet has some rubocop rules for sorbet that would be great to include.

**Comment by britneywright** [comment]
@bmulholland Which version of rubocop are you using? sorbet is in supported in rubocop-0-92
https://github.com/codeclimate/codeclimate-rubocop/pull/261

**Comment by bmulholland** [comment]
Huh, I definitely searched for sorbet in this repo before filing the issue. Not sure how I missed that!

In any case, I'm unfortunately using the airbnb channel. Would you accept a PR to add sorbet to that channel?

**Comment by britneywright** [comment]
You can try opening a PR. Keep in mind the airbnb plugin has some strict dependencies (https://rubygems.org/gems/rubocop-airbnb) that may make compatibility with other plugins difficult. That's why we had to make a separate channel to support it.

**Comment by bmulholland** [comment]
Yeah, great point. So I forked it and upgraded everything to latest, including much more open version requirements. It works fine locally on my (admittedly small) codebase; let's see what airbnb says.
